.github/workflows/openms_ci_matrix_full.yml

```diff
@@ -436,21 +545,99 @@ jobs:
+          if [[ "${{ github.ref_name }}" == "nightly" ]]; then
+            folder=nightly
+          elif [[ "${{ github.ref_name }}" == release/* ]]; then
+            folder=${{ github.ref_name }}
           else
-            folder=experimental/${{ env.RUN_NAME }}
+            folder=experimental/${{ github.ref_name }}
           fi
           echo "Uploading installers to: " $folder
 
           mkdir -p ~/.ssh/
           echo "$PASS" > ~/.ssh/private.key
           sudo chmod 600 ~/.ssh/private.key
           rsync --progress -avz -e "ssh -i ~/.ssh/private.key -o StrictHostKeyChecking=no" $GITHUB_WORKSPACE/* "$USER@$HOST:/OpenMSInstaller/${folder}"
+        
+      - name: Make installer latest
+        if: inputs.mark_as_latest
+        shell: bash
+        env:
+          PASS: ${{ secrets.ARCHIVE_RRSYNC_SSH }}
+          USER: ${{ secrets.ARCHIVE_RRSYNC_USER }}
+          HOST: ${{ secrets.ARCHIVE_RRSYNC_HOST }}
+        run: |
+          if [[ "${{ github.ref_name }}" == release/* ]]; then
+            folder=${{ github.ref_name }}
+          else
+            echo refusing to mark non-release branch as latest
+            exit 1
+          fi
+
+          mkdir -p ~/.ssh/
+          echo "$PASS" > ~/.ssh/private.key
+          sudo chmod 600 ~/.ssh/private.key
+          ln -s ./$folder latest  #create link to the release folder
+          rsync --progress -avz -e "ssh -i ~/.ssh/private.key -o StrictHostKeyChecking=no" latest "$USER@$HOST:/OpenMSInstaller/release"
+
+      - name: create RELEASE_TEXT
+        id: mk_release_txt
+        if: inputs.do_release
+        shell: bash
+        run: |
+            #TODO Handle changelog_params
+            ## Header
+            cat > releaseTextHeader.txt << EOF
+            Dear OpenMS-Users,
+
+            we are proud to announce the release of OpenMS ${{ needs.build-and-test.outputs.version_number }}. Grab it <a href="https://abibuilder.cs.uni-tuebingen.de/archive/openms/OpenMSInstaller/release/${{ needs.build-and-test.outputs.version_number }}">here</a>
+
+            In the following you find all important changes to the previous version:
+
+            EOF
 
+            ## Header
+            cat > releaseTextParamHeader.txt << EOF
+
+            Additionally, the following changes were made to parameters of our TOPP tools:
+
+            EOF
+
+            ##Footer
+            cat > releaseTextFooter.txt << EOF
+
+            Best regards,
+            The OpenMS-Developers
+
+            EOF
+            RELEASE_TEXT=$(cat releaseTextHeader.txt $GITHUB_WORKSPACE/changelog.txt releaseTextFooter.txt)
+            RELEASE_TEXT_ESCAPED_QUOTES="${RELEASE_TEXT//\"/\\\"}"
+            cat releaseTextHeader.txt $GITHUB_WORKSPACE/changelog.txt releaseTextFooter.txt > RELEASE_TEXT_GH.md
+            echo release_text_contents=`base64 -w0 RELEASE_TEXT_GH.md` >> $GITHUB_OUTPUT
+      
+      - name: run GHA release action.
```

Comment:
Could you test this step? I can't find a release for 2.9.1.
Or do you delete them quickly always?

Reply:
Yeah, I've been deleting them.